sklearn/linear_model/tests/test_logistic.py

```diff
@@ -78,6 +80,7 @@ def test_predict_2_classes():
                                          random_state=0), X_sp, Y1)
 
 
+@pytest.mark.filterwarnings('ignore: Default solver will be changed')  # 0.22
```

Comment:
Why are these required here? That's a bit confusing to me.

Reply:
I mean why has this become necessary with your change and not before?

Reply:
It's quite mysterious for me too, the test passes without the `filterwarning` on my machine but not on travis